engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,16 +162,22 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRot;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                viewRot = new Quat4f(TeraMath.DEG_TO_RAD * lookYaw, 0, 0);
-                viewRot.rotate(relMove, relMove);
+                if (!playerCamera.isTracked()) {
```

Comment:
@Josharias That was the case before my recent change. The idea was that the `trySetOrientation` (or something) method was specific per-camera and the VR camera ignored input to that method, where a normal camera did not.

Re: keyboard/mouse controls as a form of "head tracking" - to some extent, yes, it makes sense. That is why it's more appropriate to have a separate `LocalPlayerSystem` for VR. I would honestly prefer to not do that in this pull request.

Anyway, I leave it up to the two of you to decide among these three options:
1. Camera class contains a `setOrientation`method to handle the logic of optionally using or disregarding the pose received from `LocalPlayerSystem`.
2. The current state.
3. Scrap this PR until I do more work on a new player system for VR.